signer/src/bitcoin/validation.rs

```diff
@@ -833,11 +843,11 @@ pub enum WithdrawalRequestStatus {
     /// block anchoring the Stacks block that confirmed the withdrawal
     /// request, and the block hash is the associated block hash of that
     /// bitcoin block.
-    Confirmed(u64, BitcoinBlockHash),
```

Comment:
You removed the block height and block hash, but left their descriptions in the comments.

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/1344/commits/d083cfce70ebc4887a68e1b442094cfa66ff41ff.